compat/server.browser.js

```diff
@@ -1,4 +1,11 @@
+import { renderToString as preactRenderToString } from 'preact-render-to-string';
```

Comment:
nit: No need for the `as`, just keep it as `renderToString`

Reply:
done